lib/painted_rabbit/serializer.rb

```diff
@@ -0,0 +1,16 @@
+class PaintedRabbit::Serializer
+  def initialize(block)
+    @block = block
+  end
+  def self.serialize(&block)
+    @_blah = self.new(block)
+  end
+
+  def self.bleh
```

Comment:
NB: Can we replace this metasyntactic variable with a name representing what's actually happening?